core/bin/zksync_api/src/bin/dev-liquidity-token-watcher.rs

```diff
@@ -94,28 +97,51 @@ fn load_tokens(path: impl AsRef<Path>) -> Vec<(String, String)> {
     tokens
 }
 
-#[derive(Debug, Serialize, Deserialize)]
-struct GrqaphqlQuery {
-    query: String,
+async fn handle_get_asset_platforms(storage: web::Data<CoinGeckoStorage>) -> Result<HttpResponse> {
+    let keys: Vec<PlatformId> = storage.keys().map(|key| key.clone()).collect();
+    if !keys.is_empty() {
+        return Ok(HttpResponse::Ok().json(vec![AssetPlatform {
+            id: keys[0].clone(),
+            chain_identifier: Some(9999),
+            name: format!("Local Dev"),
+            shortname: format!("localdev"),
+        }]));
+    }
+
+    HttpResponse::InternalServerError().message_body(AnyBody::from_message("Nothing in storage"))
 }
 
-async fn handle_graphql(
-    params: web::Json<GrqaphqlQuery>,
-    volume_storage: web::Data<VolumeStorage>,
+async fn handle_get_coin_contract(
```

Comment:
this function basically returns the `default_volume` from the config?

Reply:
correct